### ISSUE TITLE
Backbone training example?

### ISSUE THREAD

**Issue by temp3rr0r** [DataDog/toto#42]
Hello!

First of all, thank you for releasing the source code for TOTO. It is a very impressive model!

Is it possible to share a small example training the backbone model?

I have been trying to reverse engineer an example (using the hyperparameters from your well-written paper) but unfortunately I think I am doing something wrong with the composite loss function.

Thank you in advance!

With kind regards,
Kostas
